src/include/sof/drivers/ipc.h

```diff
@@ -47,6 +47,9 @@ struct ipc_msg;
 #define COMP_TYPE_BUFFER	2
 #define COMP_TYPE_PIPELINE	3
 
+/** \brief IPC send timeout in microseconds. */
+#define IPC_TIMEOUT	100
```

Comment:
.. here for description is better to add 'scheduling period for IPC task'

Reply:
I changed it to 'scheduling period for IPC task in microseconds' to indicate the time unit.